ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -699,7 +692,8 @@ module Winbind = struct
   let random_string len =
     let upper_char_start = 65 in
     let upper_char_len = 26 in
-    String.init len (fun _ -> upper_char_start + Random.int upper_char_len |> char_of_int)
+    String.init len (fun _ ->
```

Comment:
I would like it better if you implemented a `rand_char: unit -> char` and used that in `String.init`